databases_test.go

```diff
@@ -533,6 +533,45 @@ func TestDatabases_DeleteUser(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestDatabases_ResetUserAuth(t *testing.T) {
+	setup()
+	defer teardown()
+	dbID := "deadbeef-dead-4aa5-beef-deadbeef347d"
+	path := fmt.Sprintf("/v2/databases/%s/users/user/reset_auth", dbID)
+
+	body := `
+{
+  "user": {
+    "name": "name",
+    "role": "foo",
+    "password": "pass",
+	"mysql_settings": {
+		"auth_plugin": "caching_sha2_password"
```

Comment:
Can you make the indentation consistent here? It should be two spaces, like the rest of this object.

Reply:
good catch thanks